packages/broken-link-checker/src/run.js

```diff
@@ -15,9 +15,16 @@ const run = ({ server, port, config, checkerOptions }) => {
     html: (/* tree, robots, response, pageUrl, customData */) => {},
     junk: (/* link, customData */) => {},
     link: (link /* , customData */) => {
+      if (!link.internal) {
+        const { protocol } = new URL(link.url.original);
+        if (protocol === 'http:') {
+          console.warn(`Unsecure external link : ${link.url.original}`);
```

Comment:
```suggestion
          console.warn(`Insecure external link : ${link.url.original}`);
```
One of the illogical parts of english: There is no "unsecure", but there is "unsecured" and there there  is "insecure" but no "insecured".   Don't ask, it's as explainable as most of german :-)

Reply:
Ah right, thanks :)